src/test/java/teammates/common/datatransfer/attributes/NotificationAttributesTest.java

```diff
@@ -0,0 +1,269 @@
+package teammates.common.datatransfer.attributes;
+
+import java.time.Instant;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.NotificationTargetUser;
+import teammates.common.datatransfer.NotificationType;
+import teammates.storage.entity.Notification;
+import teammates.test.BaseTestCase;
+
+/**
+ * SUT: {@link NotificationAttributes}.
+ */
+public class NotificationAttributesTest extends BaseTestCase {
+    private static final Instant STARTTIME_ONE = Instant.now().plusSeconds(3600);
+    private static final Instant ENDTIME_ONE = Instant.now().plusSeconds(7200);
+    private static final Instant STARTTIME_TWO = Instant.now().plusSeconds(1000);
+    private static final Instant ENDTIME_TWO = Instant.now().plusSeconds(10000);
```

Comment:
I feel we don't need to declare these constants. I believe your intention is to reduce duplicates, but it's probably okay just to say `NotificationAttributes.builder("notificationId").withStartTime(Instant.now().plusSeconds(3600))` in the test function itself, which is more explicit.